docs/en/connector-v2/source/SftpFile.md

```diff
@@ -94,6 +94,7 @@ The File does not have a specific type list, and we can indicate which SeaTunnel
 | compress_codec            | String  | No       | None                | The compress codec of files and the details that supported as the following shown: <br/> - txt: `lzo` `None` <br/> - json: `lzo` `None` <br/> - csv: `lzo` `None` <br/> - orc: `lzo` `snappy` `lz4` `zlib` `None` <br/> - parquet: `lzo` `snappy` `lz4` `gzip` `brotli` `zstd` `None` <br/> Tips: excel type does Not support any compression format                            |
 | archive_compress_codec    | string  | no       | none                |
 | encoding                  | string  | no       | UTF-8               |
+| null_format               | string  | no       | -                   | Only used when file_format_type is text. null_format to define which strings can be represented as null.                                                                                                                                                                                                                                                                        |
```

Comment:
provide an example? E.g. `\N`

Reply:
example
https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=27362034#LanguageManualDDL-StorageFormatsStorageFormatsRowFormat,StorageFormat,andSerDe

https://github.com/alibaba/DataX/blob/master/hdfsreader/doc/hdfsreader.md

Reply:
I mean put an example into the doc. Provide some common null format as example.

Reply:
fixed